sites/partners/src/components/listings/PaperListingForm/sections/BuildingSelectionCriteria.tsx

```diff
@@ -225,11 +224,83 @@ const BuildingSelectionCriteria = () => {
       </Grid>
 
       <Drawer
-        open={drawerState}
-        title={t("listings.addBuildingSelectionCriteria")}
+        isOpen={drawerState}
         onClose={() => resetDrawerState()}
-        ariaDescription="Form with photo upload dropzone"
-        actions={[
+        ariaLabelledBy="building-selection-criteria-drawer-header"
+        ariaDescribedBy="building-selection-criteria-drawer-description"
+      >
+        <Drawer.Header id="building-selection-criteria-drawer-header">
+          {t("listings.addBuildingSelectionCriteria")}
+        </Drawer.Header>
+        <Drawer.Content>
+          <p id="building-selection-criteria-drawer-description" className="sr-only">
+            Form with PDF upload dropzone
```

Comment:
Even though these are sr-only I think we should keep content tracked to the strings files

Reply:
Yeah…and I wonder if we even need that particular descriptive text (it never worked before, anyway)